### PR TITLE
Weapon Powercells: Spritercode edition

### PR DESCRIPTION
**Pull request by triplezeta**
## About The Pull Request

image

New weapon powercell sprites! 

Ports the cell recharger resprite by Azlan#0852, from https://github.com/BeeStation/BeeStation-Hornet/pull/2339

Adds the large-sized weapon powercell, a large powercell meant for use in similarly large weapons. Holds 50 MJ and can be unlocked from Advanced weapon tech. 

## Why It's Good For The Game

good sprites good

## Changelog
:cl: Azlanon & Triplezeta
add: Large weapon powercells, for use in large weapons.
imageadd: new weapon powercell sprites, and Azlan's cell charger resprite
/:cl:

### CODE REVIEW

**Comment by Baldir-Odensen** [comment]
Ok so the larger one does it hold the curve of the other ones of costing as much as the next step up

**Comment by triplezeta** [comment]
it costs a bit more than a bluespace powercell

**Review by Superyodeler — APPROVED** [review]

**Comment by Superyodeler** [comment]
Thicc weapon powercell

File: waspstation/code/modules/power/cell.dm [hunk 1]
```diff
@@ -15,12 +15,16 @@
 	cut_overlays()
 	if(grown_battery)
 		. += mutable_appearance('icons/obj/power.dmi', "grown_wires")
-	if(charge < 0.01)
+	if(charge < 0.1)
```
Comment (MarkSuckerberg) [inline waspstation/code/modules/power/cell.dm:18]: You sure about this?

Comment (triplezeta) [reply waspstation/code/modules/power/cell.dm:18]: Yes.

**Pull request merged**